metabase/metabase
# Entity Search + Indexing

Resolves https://github.com/metabase/metabase/issues/29819
Resolves https://github.com/metabase/metabase/issues/30167

### Description

Allows model columns to be indexed and searched globally 🥳 

### How to verify

Allow Indexes on a column by toggling the toggle in the model metadata sidebar

![Screen Shot 2023-04-13 at 12 06 30 PM](https://user-images.githubusercontent.com/30528226/231846314-47e47415-9651-447f-a58d-da9abcf10284.png)

Global search should surface items

![Screen Shot 2023-04-13 at 12 06 03 PM](https://user-images.githubusercontent.com/30528226/231846367-cb6d55d7-c286-4f37-a11e-9c4a8c9b504b.png)

> **Note**
> you may encounter apparently "broken" links when clicking on index items. this is a limitation with our object detail view which I'll be working on more in #30168 



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30082)
